tests/integration/relation_tests/legacy_relations/helpers.py

```diff
@@ -136,7 +136,7 @@ async def mongo_tls_command(ops_test: OpsTest) -> str:
     replica_set_uri = f"mongodb://{hosts}/admin?replicaSet={app}"
 
     return (
-        f"charmed-mongodb.mongo '{replica_set_uri}'  --eval 'rs.status()'"
+        f"charmed-mongodb.mongosh '{replica_set_uri}'  --eval 'rs.status()'"
```

Comment:
I see we use this in many places, this looks like a good candidate to be extracted to a constant. 

Maybe we can do that as a part of this PR? What do you think?

Reply:
yes, please
```suggestion
        f"{MONGO_SHELL} '{replica_set_uri}'  --eval 'rs.status()'"
```